.github/ISSUE_TEMPLATE.md

```diff
@@ -24,4 +24,8 @@ Describe what you expect the output to be. Knowing the correct behavior is also
 Provide any additional information here.
 
 #### Current Version:
+<<<<<<< HEAD
```

Comment:
A lot of files now have these conflicts

Reply:
I found only 3 files and fixed the conflict.